uint16[] id_list

bool[] torque
bool[] error
bool[] ping
string[] mode

uint16 CONTROL_PWM = 0
uint16 CONTROL_CURRENT = 1
uint16 CONTROL_VELOCITY = 2
uint16 CONTROL_POSITION = 3
uint16 CONTROL_EXTENDED_POSITION = 4
uint16 CONTROL_CURRENT_BASED_POSITION = 5